Promact.Oauth.Server/src/Promact.Oauth.Server/Consent/ConsentOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Promact.Oauth.Server.Consent
+{
+    public class ConsentOptions
+    {
+        public static bool EnableOfflineAccess = true;
+        public static string OfflineAccessDisplayName = "Offline Access";
```

Comment:
Moved this ConsentOptions's string to StringConstant and keep in ConsentOptions region